apps/demo/translations/src/en/fields.json

```diff
@@ -90,5 +90,22 @@
     "options": {
       "null": "Select..."
     }
+  },
+  "saveForm": {
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "saveEmail": {
+    "label": "Enter your email address"
+  },
+  "saveRef": {
+    "label": "Create a memorable reference for your form",
```

Comment:
What is your reference for this form?

Reply:
changed line 108 to 'What is your reference for this form?'